### PR TITLE
[FireEye HX] fetch-incidents command did not fetch more than 50 incidents

### PR DESCRIPTION
**Pull request by israelpoli**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/XSUP-20982

## Description
There was a limit on the fetch-command so that it did not bring more than 50 incidents.
Additionally, changed the timeout for polling commands in the integration and now the timeout is 30 minutes.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FireEyeHX/ReleaseNotes/2_3_2.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### FireEye Endpoint Security (HX) v2
+- Fixed an issue where ***fetch-incidents*** command did not fetch more than 50 incidents.
+- Fixed an issue where the ***fireeye-hx-file-acquisition*** command failed due timeout, from now the timeout is 30 minutes.
```
Comment (dorschw) [inline Packs/FireEyeHX/ReleaseNotes/2_3_2.md:4]:
```suggestion
- Fixed an issue where the ***fireeye-hx-file-acquisition*** command timed out after 10 minutes, by updating the limit to 30 minutes.
```

File: Packs/FireEyeHX/ReleaseNotes/2_3_2.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### FireEye Endpoint Security (HX) v2
+- Fixed an issue where ***fetch-incidents*** command did not fetch more than 50 incidents.
```
Comment (dorschw) [inline Packs/FireEyeHX/ReleaseNotes/2_3_2.md:3]:
```suggestion
- Fixed an issue where ***fetch-incidents*** command would not fetch more than 50 incidents at a time.
```

**Review by dorschw — APPROVED** [review]

File: Packs/FireEyeHX/ReleaseNotes/2_3_2.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#### Integrations
+##### FireEye Endpoint Security (HX) v2
+- Fixed an issue where ***fetch-incidents*** command would not fetch more than 50 incidents at a time.
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/ReleaseNotes/2_3_2.md:3]:
```suggestion
- Fixed an issue where the ***fetch-incidents*** command would not fetch more than 50 incidents at a time.
```

File: Packs/FireEyeHX/ReleaseNotes/2_3_2.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
...
+- Fixed an issue where the ***fireeye-hx-file-acquisition*** command failed due timeout, from now the timeout is 30 minutes (previously 10 minutes).
```
Comment (ShirleyDenkberg) [inline Packs/FireEyeHX/ReleaseNotes/2_3_2.md:4]:
```suggestion
- Fixed an issue where the ***fireeye-hx-file-acquisition*** command failed due to a timeout. Updated the timeout to 30 minutes (previously it was 10 minutes).
```

**Comment by ShirleyDenkberg** [comment]
@dorschw Doc review completed.

**Pull request merged**